api/routes/candidatesRouter.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express');
+
+const router = express.Router();
+
+const Candidates = require('../models/candidates-model');
+
+router.get('/', async (req, res) => {
+  try {
+    const candidates = await Candidates.getCandidates();
+    if (candidates.length > 0) {
+      res.status(200).json(candidates);
+    } else {
+      res.status(404).json({ message: 'No candidates found in the database' });
+    }
+  } catch (err) {
+    res.status(500).json(err);
+  }
+});
+
+router.get('/:id', async (req, res) => {
+  const { id } = req.params;
+  try {
+    const candidate = await Candidates.getCandidatesById(id);
+    if (candidate) {
+      res.status(200).json(candidate);
+    } else {
+      res.status(404).json({ message: 'No candidate found with that ID' });
+    }
+  } catch (err) {
+    res.status(500).json(err);
+  }
+});
+
+router.post('/', async (req, res) => {
+  Candidates.addCandidate(req.body).then(error => {
```

Comment:
I believe this should be .then(response object=>{
res..etc(response object)}).catch(err=>{ etc ...res..etc(err)})